client/signup/steps/choose-service/index.tsx

```diff
@@ -0,0 +1,143 @@
+import { isEnabled } from '@automattic/calypso-config';
+import {
+	getPlan,
+	PLAN_PREMIUM,
+	PLAN_WPCOM_PRO,
+	WPCOM_DIFM_LITE,
+} from '@automattic/calypso-products';
+import { IntentScreen } from '@automattic/onboarding';
+import { useTranslate } from 'i18n-calypso';
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import QueryProductsList from 'calypso/components/data/query-products-list';
+import { preventWidows } from 'calypso/lib/formatting';
+import StepWrapper from 'calypso/signup/step-wrapper';
+import {
+	getProductDisplayCost,
+	isProductsListFetching,
+} from 'calypso/state/products-list/selectors';
+import { saveSignupStep, submitSignupStep } from 'calypso/state/signup/progress/actions';
+import { mouse, headset } from '../../icons';
+import { ChoiceType } from './types';
+import type { SelectItem } from '@automattic/onboarding';
+
+import './style.scss';
+
+interface Props {
+	existingSiteCount: number;
+	goToNextStep: () => void;
+	submitSignupStep: ( { stepName, wasSkipped }: { stepName: string; wasSkipped: boolean } ) => void;
+	goToStep: ( stepName: string ) => void;
+	stepName: string;
+	queryObject: {
+		siteSlug?: string;
+		siteId?: string;
+	};
+}
+
+type DIFMOrBuiltByIntent = SelectItem< ChoiceType >;
+
+const Placeholder = () => <span className="choose-service__placeholder">&nbsp;</span>;
+
+export default function ChooseServiceStep( props: Props ): React.ReactNode {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+	const displayCost = useSelector( ( state ) => getProductDisplayCost( state, WPCOM_DIFM_LITE ) );
+	const isLoading = useSelector( isProductsListFetching );
+
+	const headerText = translate( 'Let our experts create your dream site' );
+
+	const subHeaderText = translate(
+		'Get your business up quickly with our experts building your responsive, professional website. Select your best option.'
+	);
+
+	const intents: DIFMOrBuiltByIntent[] = [
+		{
+			key: 'difm',
+			title: translate( 'Do It For Me' ),
+			description: translate(
+				'Get a professionally designed, mobile-optimized website in %(fulfillmentDays)d business days or less for a one-time fee of {{PriceWrapper}}%(displayCost)s{{/PriceWrapper}} plus a one year subscription of the %(plan)s plan.',
+				{
+					args: {
+						displayCost,
+						fulfillmentDays: 4,
+						plan: isEnabled( 'plans/pro-plan' )
```

Comment:
I'm not entirely sure, but should we use [isEligibleForProPlan](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/plans-comparison/is-eligible-for-pro-plan.ts) here instead?

Reply:
Good spot! Since the Premium plan has been retired, I think that we can remove this entire check and just have `getPlan( PLAN_WPCOM_PRO )?.getTitle()` here instead.